fiftyone/operators/delegated.py

```
@@ -838,61 +855,69 @@ async def _execute_operator(self, doc):
+                # Resolve output types now
+                ctx.request_params["target"] = "outputs"
+                ctx.request_params["results"] = result
+
+                outputs = await resolve_type_with_context(operator, ctx)
+                if outputs is not None:
+                    outputs_schema = outputs.to_json()
+            except (AttributeError, Exception):
+                logger.warning(
+                    "Failed to resolve output schema for the operation."
+                    + traceback.format_exc(),
+                )
```

Pylint (4.0.2)

[warning] Use lazy % formatting in logging functions

(W1201)